README.md

```diff
@@ -14,15 +14,15 @@ Do contribute if you find a well written article, a good tutorial, playlist or a
 
 ### Open Source
 
-- [https://medium.freecodecamp.org/a-beginners-very-bumpy-journey-through-the-world-of-open-source-4d108d540b39](https://medium.freecodecamp.org/a-beginners-very-bumpy-journey-through-the-world-of-open-source-4d108d540b39)
-- [https://www.hackerearth.com/getstarted-opensource/](https://www.hackerearth.com/getstarted-opensource/)
-- [https://github.com/facebook/react/issues?q=is%3Aissue+is%3Aopen+label%3A%22Difficulty%3A+beginner%22](https://github.com/facebook/react/issues?q=is%3Aissue+is%3Aopen+label%3A%22Difficulty%3A+beginner%22)
-- [https://medium.freecodecamp.com/@shubheksha](https://medium.freecodecamp.com/@shubheksha)
-- [https://github.com/github/opensource.guide/issues/143](https://github.com/github/opensource.guide/issues/143)
-- [https://nandomoreira.me/open-source-guide/how-to-contribute/](https://nandomoreira.me/open-source-guide/how-to-contribute/)
-- [https://opensource.guide/](https://opensource.guide/)
-- [https://codeburst.io/what-is-open-source-and-why-is-it-so-popular-5f6b0eaf819a](https://codeburst.io/what-is-open-source-and-why-is-it-so-popular-5f6b0eaf819a)
+- [A bumpy journey through the world of open source](https://medium.freecodecamp.org/a-beginners-very-bumpy-journey-through-the-world-of-open-source-4d108d540b39) - A beginner's tale
+- [Hackerearth](https://www.hackerearth.com/getstarted-opensource/) - Get started with open source
```

Comment:
You did a good job! But you may have gotten the idea a bit wrong. The titles of the articles are what matter not the publisher.

For instance, it should be:
[Get started with open source](https://www.hackerearth.com/getstarted-opensource/)
Not:
[Hackerearth](https://www.hackerearth.com/getstarted-opensource/) - Get started with open source

In case of ambiguity you can even add (not actually necessary): 
[Get started with open source](https://www.hackerearth.com/getstarted-opensource/) - Hackerearth

Reply:
I will do it.